style.css

```diff
@@ -0,0 +1,454 @@
+.media-links {
```

Comment:
- [ ] If you notice, there is a white border all around your page which is not supposed to be there, so it would be nice if you can remove it from the page itself so you stick to the design as much as possible.